src/components/QuestionTracker/QuestionTracker.module.css

```diff
@@ -0,0 +1,34 @@
+.container {
+  display: flex;
+  width: fit-content;
+  align-items: center;
+  margin: 0rem auto;
+  flex-wrap: wrap;
+  cursor: pointer;
+}
+
+.question_no_wrapper {
+  padding: 0.3rem 0.8rem;
+}
+
+.attempted {
+  background-color: var(--primary-color);
+  width: 100%;
+  height: 1.3rem;
+}
+
+.non_attempted {
+  width: 100%;
+  height: 1.3rem;
+}
+
+.tracker_container {
+  background-color: rgba(255, 255, 255, 0.15);
```

Comment:
color should be consistent either use rgb or variables or hex. preferred would be hex